lib/pod/command/plugins/installed.rb

```diff
@@ -0,0 +1,52 @@
+
+module Pod
+  class Command
+    class Plugins
+      # The list subcommand. Used to list all known plugins
+      #
+      class Installed < Plugins
+        self.summary = 'List plugins installed on your machine'
+        self.description = <<-DESC
+                List all installed plugins and their
+                respective version
+        DESC
+
+        def self.options
+          # Silent mode is meaningless for this command as
+          # the command only purpose is to print information
+          super.reject { |option, _| option == '--silent' }
+        end
+
+        def run
+          plugins = CLAide::Command::PluginManager.specifications
+          UI.title 'Installed CocoaPods Plugins:' do
+            if self.verbose?
```

Comment:
o_O why did I write that? Too much holidays away from Ruby, I suppose :laughing: